plone/app/discussion/setuphandlers.py

```diff
@@ -0,0 +1,59 @@
+from plone.base.interfaces import INonInstallable
+from Products.CMFCore.utils import getToolByName
+from zope.interface import implementer
+
+
+DEFAULT_TYPES = [
+    "Document",
+    "Event",
+    "File",
+    "Folder",
+    "Image",
+    "News Item",
+    "Collection",
+    "Link",
+]
+
+BEHAVIOR = "plone.allowdiscussion"
+
+
+@implementer(INonInstallable)
+class HiddenProfiles:
+    def getNonInstallableProfiles(self):
+        return [
+            "plone.app.contenttypes:to_3000",
```

Comment:
Should this be rather `plone.app.discussion:to_3000` ? But the profile you created is rather `to_2002`. Though it is not registered per se, but rather a `to_3000`.

Is that probably some copy and pasting leftovers? 😄 Otherwise I'm a bit confused between the `to_2002` and `to_3000` 🙃

Reply:
Oops, some copy paste leftovers indeed.

And profile `to_3000` in this PR is a copy of `to_2002` in my other PR for branch 4.x.  And I apparently forgot to rename the directory.  And I see that GenericSetup does not mind at all that the directory does not exist.  I thought I had tested this properly yesterday, but something was lacking.

Fixed now.